name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/ClientEditorSidebar.java

```diff
@@ -213,23 +213,25 @@ public void drop(DropTargetEvent event)
             {
                 if (SecurityTransfer.getTransfer().isSupportedType(event.currentDataType))
                 {
-                    Security security = SecurityTransfer.getTransfer().getSecurity();
-                    if (security != null)
-                    {
-                        // if the security is dragged from another file, add
-                        // a deep copy to the client's securities list
-                        if (!editor.getClient().getSecurities().contains(security))
-                        {
-                            security = security.deepCopy();
-                            editor.getClient().addSecurity(security);
+                    List<Security> securities = SecurityTransfer.getTransfer().getSecurities();
+                    if (securities != null)
+                    {   
+                        for(Security security : securities) {
```

Comment:
If you are using Eclipse, then the default formatter for this project should be configured by default. If you run *ctrl-f* it also should format the file. It would help me if the code is formatted with the project formatter because then later commits do not include formatting changes and are easier to read. And here the curly brace would be on the next line. Just a minor thing...